assets/src/edit-story/components/inspector/karma/prepublishSelect.karma.js

```diff
@@ -241,5 +241,75 @@ describe('Pre-publish checklist select offending elements onClick', () => {
         tooSmallLinkElement.id
       );
     });
+
+    it('should open the design inspector panel and focus the text input', async () => {
+      await fixture.act(() => {
+        insertElement('image', {
+          x: 0,
+          y: 0,
+          width: 640 / 2,
+          height: 529 / 2,
+          resource: {
+            type: 'image',
+            mimeType: 'image/jpg',
+            src: 'http://localhost:9876/__static__/earth.jpg',
+          },
+        });
+      });
+      await openPrepublishPanel();
+      await openRecommendedPanel();
+      const imageMissingAltTextRow = fixture.screen.getByText(
+        MESSAGES.ACCESSIBILITY.MISSING_IMAGE_ALT_TEXT.MAIN_TEXT
+      );
+      await fixture.events.mouse.clickOn(imageMissingAltTextRow);
+
+      expect(
+        fixture.editor.inspector.designPanel.node.contains(
+          document.activeElement
```

Comment:
Could this be more specific to reference the element that was created and clicked on? It seems that through this test it'd never test that the element was actually selected since it would start selected I think.

Reply:
Maybe I'm misinterpreting... This is the element that should be focused after you click the prepublish checklist.